.github/workflows/cd-publish.yml

```diff
@@ -3,38 +3,98 @@ name: CD - Publish for Blazor / FluentUI
 on:
   workflow_dispatch
 
+env:
+  OUTPUT_PATH: ${{ github.workspace }}/.nupkg
+  NUGET_SOURCE: "https://api.nuget.org/v3/index.json"
+  DOTNET_VERSION: "5.0.5"
+  GITHUB_SOURCE: "https://nuget.pkg.github.com/Azure/index.json"
```

Comment:
I don't know what GITHUB_SOURC is but the value doesn't match since we're not Azure.

Reply:
that a link to the GitHub artifact

Reply:
What does this do and why do we need it?

Reply:
GITHUB_SOURCE: "https://nuget.pkg.github.com/Azure/index.json" is removed since we are not publishing artifacts to download it. 
NUGET_SOURCE: "https://api.nuget.org/v3/index.json"
is moved to nuget.config file; no need to duplicate